README.md

```diff
@@ -1,3 +1,7 @@
+# DEPRECATED
+
+This repository is no longer maintained. It is replaced by the [GOV.UK Developer Docs](https://docs.publishing.service.gov.uk/manual/setting-up-new-rails-app.html#header).
```

Comment:
This page links back to this repository? https://docs.publishing.service.gov.uk/manual/setting-up-new-rails-app.html#bootstrapping-a-new-application

Reply:
There's a corresponding PR to update (https://github.com/alphagov/govuk-developer-docs/pull/1054). I wasn't planning to merge this one until the new page is in place :-).